src/menu/menu.scss

```diff
@@ -47,7 +47,7 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
   cursor: pointer;
   box-sizing: border-box;
   line-height: normal;
-
+  width: auto;
```

Comment:
I don't think you really need this width, it should come from the flexbox if you use `iui-content`.

Curious how you ended up with this solution. If you try using our `MenuItem` react component, it will not let you use it without wrapping the children with `iui-content`.

Reply:
this has been removed